### PR TITLE
Export 3

### PR DESCRIPTION
**Pull request by nono**

### CODE REVIEW

File: src/export.coffee [hunk 1]
```diff
@@ -46,6 +46,26 @@ getContent = (couchClient, binaryId, type, callback) ->
...
+    couchClient.get u, (err, res, body) ->
+        if err?
+            done err
+        else
+            async.eachSeries body.rows, (row, cb) ->
+                forEach row.doc, cb
+            , (err) ->
+                if err
+                    done err
+                else if body.rows.length == limit && offset < 10000
```
Comment (aenario) [inline src/export.coffee]: shouldn't we at least log an error for the too many document cases ?

File: src/export.coffee [hunk 2]
```diff
@@ -46,6 +46,26 @@ getContent = (couchClient, binaryId, type, callback) ->
             stream.on 'error', (err) -> log.error err
             callback null, stream
 
+# Note: using offset is accidentally quadratic (sic)
```
Comment (aenario) [inline src/export.coffee]: [nitpick] Why not `?start_key=body.rows[limit]._id`?

File: src/export.coffee (hunk 1 above)
Comment (nono) [reply src/export.coffee]: Oops, it was for debugging and I forgot to remove it

File: src/export.coffee (hunk 2 above)
Comment (nono) [reply src/export.coffee]: Well, you got again the last document in the results of the new request. And there is no `include_start=false` parameter

**Comment by nono** [comment]
@aenario I think this PR is ready to be merged. And if you can also publish a new version of cozy-monitor, it would be nice

**Pull request merged**